src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -294,7 +294,7 @@ public static class Dimension
          public const string MOLAR_CONCENTRATION = "Concentration (molar)";
          public const string MASS_CONCENTRATION = "Concentration (mass)";
          public const string DIMENSIONLESS = "Dimensionless";
-         public const string AMOUNT = "Amount";
+         public const string MOLAR_AMOUNT = "Amount";
```

Comment:
THis is nicer when matching MASS To MOLAR. All molar starts with Molar, all MASS start with MASS